codegen/api-specs/nuts-storage-api-v1.yaml

```diff
@@ -34,15 +75,19 @@ paths:
               schema:
                 $ref: '#/components/schemas/SecretResponse'
         "404":
-          description: The lookup failed, the secret for the provided path could not be found.
+          description: The lookup failed, the secret for the provided key could not be found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
         "400":
-          description: The request was invalid. Check the error response for more details.
+          description: The lookup failed. Check the error response for more details.
```

Comment:
400 is a bit weird here. 400s are returned when the client input is wrong, after changing the input the request may be tried again. This request doesn't have client input apart from the path, which always returns a 404 when incorrect?